### PR TITLE
hoomd

### PR DESCRIPTION
**Pull request by joaander**
HOOMD is a python library I develop for particle simulations in soft matter research: http://glotzerlab.engin.umich.edu/hoomd-blue/

I have previously released this software on my own anaconda channel, and would now like to add it to conda-forge to increase availability.

### CODE REVIEW

**Comment by joaander** [comment]
My apologies for the number of commits it required to build correctly. There are a number of differences between the CI's build environment and my local test rig.

This recipe is complete and ready to be reviewed. It follows the same form as my other recent package: pull request #6047

**Comment by joaander** [comment]
Hi @conda-forge/staged-recipes

I just noticed in the README that I should explicitly ping you to request a review of a PR that has been passing for more than a week.

**Comment by scopatz** [comment]
This looks great to me, but unfortunately there is a block on merging all PRs right now due to failure to create feedstocks. We'll get to this as soon as it is cleared up!

**Comment by joaander** [comment]
Thanks for looking at this, I wasn't aware of the block. No worries.

**Comment by scopatz** [comment]
Yeah it sucks. Some of my packages are blocked too 😉

File: recipes/hoomd/build.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+cmake ../ \
+      -DCMAKE_INSTALL_PREFIX=${SP_DIR} \
+      -DPYTHON_EXECUTABLE=${PYTHON} \
+      -DENABLE_MPI=off \
+      -DENABLE_CUDA=off \
+      -DBUILD_TESTING=off \
+      -DENABLE_TBB=on \
+      -DBUILD_JIT=on \
+
```
[the remark this answers is not in the document]
Comment (joaander) [reply recipes/hoomd/build.sh]: Yes, there is. In the most recent commit, I added a run of HOOMD's shortest batch of unit tests. Adding this was useful in that a failing test uncovered a missing dependency. However, HOOMD is a large project with extensive testing and the tests add 5-10 minutes per python version in the build on top of the already long build time. This resulted in the Travis Mac build running out of time: ``The job exceeded the maximum time limit for jobs, and has been terminated.``

How do you recommend that I proceed?

File: recipes/hoomd/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  version: "{{ version }}"
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://glotzerlab.engin.umich.edu/Downloads/{{ name }}/{{ name }}-v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
```
[the remark this answers is not in the document]
Comment (joaander) [reply recipes/hoomd/meta.yaml]: The upstream package does not support windows. I added a comment mentioning this in the latest commit.

**Comment by scopatz** [comment]
Let us know when this is ready for another round of review!

**Comment by joaander** [comment]
It's ready for review now. The latest commit was able to build and test within the travis build time limit.

**Pull request merged**